addons/calendar/wizard/calendar_provider_config.xml

```diff
@@ -14,8 +14,8 @@
                         Synchronize your calendar with Google Calendar
                     </div>
                     <group>
-                        <field name="cal_client_id" attrs="{'required': [('external_calendar_provider', '=', 'google')]}"/>
-                        <field name="cal_client_secret" password="True" attrs="{'required': [('external_calendar_provider', '=', 'google')]}"/>
+                        <field name="cal_client_id" string="Client ID" attrs="{'required': [('external_calendar_provider', '=', 'google')]}"/>
+                        <field name="cal_client_secret" string="Client Secret" password="True" attrs="{'required': [('external_calendar_provider', '=', 'google')]}"/>
```

Comment:
Could you please add this for Outlook as well?

Reply:
@flch-odoo updated for outlook